server/game/cards/13.3-PoS/UnexpectedGuile.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+
+class UnexpectedGuile extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction(card => card.isShadow() && card.controller === this.controller);
```

Comment:
By itself, this would allow it to be attached to non-character cards. Unfortunately there's no way to fill in defaults when it's a restriction function instead of a restriction matcher object (which gets the defaults merged in).